Detectors/TRD/workflow/src/TRDTrackletTransformerSpec.cxx

```diff
@@ -35,6 +35,13 @@ void TRDTrackletTransformerSpec::run(o2::framework::ProcessingContext& pc)
 {
   LOG(INFO) << "Running tracklet transformer";
 
+  TrackletTransformer transformer(mTimestamp);
```

Comment:
Why do you create now a `TrackletTransformer` object each time run is called? I would keep it as a member of TRDTrackletTransformerSpec.

Reply:
I wasn't sure how to actually implement that since the constructor takes an argument. I made an attempt. Let me know if it looks alright.

Reply:
Since your constructor argument has a default value this does not matter

Reply:
Well `mTimestamp` is not within the scope of the TrackletTransformer. I think it is working how I implemented it though

Reply:
I don't get why you need to know the time stamp at the time you create an `TrackletTransformer` object. I would not put the function which connects to the CCDB inside the constructor. Instead you can just call that function from the DPL device and then you would also get an error message if that fails in the place one would expect it ;)

Reply:
True, but there are many cases where the TrackletTransformer is used outside of the workflow context. With this change, one would need to manually call the function to load the CCDB object.

Reply:
In which cases for example? In any case one would need to load the CCDB object then, sure. But I would see this as a good thing, because that way you make sure that the user is aware that a calibration object is needed and a timestamp should be provided. A default time stamp would hide that

Reply:
Okay, I went ahead and made this change.